diff -rupN snowballz.old/lib/building.py snowballz/lib/building.py
--- snowballz.old/lib/building.py	2009-11-18 04:59:48.000000000 -0500
+++ snowballz/lib/building.py	2009-11-18 05:14:14.000000000 -0500
@@ -126,7 +126,7 @@ class Igloo(DynamicObjectStore):
             center_x = (self.sprite.left+self.sprite.right)/2
             # FIXME? under certain circumstances using TSCu_Comic will cause a
             # segfault.
-            self.label = Label(string,# font_name="TSCu_Comic",
+            self.label = Label(string, font_name="SerafettinCartoon",
                     x=center_x, y=self.sprite.top, anchor_x="center",
                     color=(0,0,0,255))
         else:
diff -rupN snowballz.old/lib/penguin_control.py snowballz/lib/penguin_control.py
--- snowballz.old/lib/penguin_control.py	2009-11-18 04:59:48.000000000 -0500
+++ snowballz/lib/penguin_control.py	2009-11-18 05:13:36.000000000 -0500
@@ -85,7 +85,7 @@ class PenguinControl:
                         x = self.unit_selection_sprite.right - 20,
                         y = self.unit_selection_sprite.top - 25,
                         font_size = 17,
-                        font_name = "TSCu_Comic",
+                        font_name = "SerafettinCartoon",
                         color = (100,0,0,255)))
 
         if self.num_workers:
@@ -93,7 +93,7 @@ class PenguinControl:
                         x = self.unit_selection_sprite.right - 20,
                         y = self.unit_selection_sprite.top - 90,
                         font_size = 17,
-                        font_name = "TSCu_Comic",
+                        font_name = "SerafettinCartoon",
                         color = (100,0,0,255)))
 
         for i, u_type in enumerate(["snowballer", "worker"]):
diff -rupN snowballz.old/map_editor.py snowballz/map_editor.py
--- snowballz.old/map_editor.py	2009-11-18 04:59:48.000000000 -0500
+++ snowballz/map_editor.py	2009-11-18 05:15:25.000000000 -0500
@@ -76,7 +76,7 @@ class ScriptArea(object):
 
         self.sprite.xy = self.scene.map.to_pixel(map_xy)
 
-        self.font = font.load(os.path.join('data', 'TSCu_Comic.ttf'), 13)
+        self.font = font.load(os.path.join('data', 'SerafettinCartoon.ttf'), 13)
         self.text = font.Text(self.font, str(self.id))
         self.text.x = self.sprite.x
         self.text.y = self.sprite.y
