--- snowballz/lib/terrain.py~	2009-05-12 14:35:02.000000000 -0400
+++ snowballz/lib/terrain.py	2009-05-12 15:31:25.000000000 -0400
@@ -85,12 +85,12 @@ class Terrain(object):
                         self.v_c[vx, vy, :3] = d, d, d
                     assert p.pop()
 
-        if not self.using_cache:
-            assert p.push("Save cache")
-            # Save cache if necessary
-            numpy.savez(os.path.join(cache_path, "cache.npz"), v=self.v,
-                    v_c=self.v_c, v_t=self.v_t)
-            assert p.pop()
+        #if not self.using_cache:
+        #    assert p.push("Save cache")
+        #    # Save cache if necessary
+        #    numpy.savez(os.path.join(cache_path, "cache.npz"), v=self.v,
+        #            v_c=self.v_c, v_t=self.v_t)
+        #    assert p.pop()
 
         #if ttype != "all":
             #assert p.push("Blending")
